src/Service/Validator.php

```diff
@@ -24,8 +24,10 @@ class Validator
     const MAGENTO_SUPPORTED_SERVICE_VERSIONS = [
         ServiceInterface::NAME_PHP => [
             '<=2.2.4' => '>=7.0 <7.2',        //'7.0.2|7.0.4|~7.0.6|~7.1.0',
-            '>=2.2.5 <2.3.0' => '>=7.0 <7.2', //'~7.0.13|~7.1.0',
-            '>=2.3.0' => '>=7.1 <7.3',        //'~7.1.3 || ~7.2.0',
+            '>=2.2.5 <2.2.10' => '>=7.0.13 <7.2', //'~7.0.13|~7.1.0',
```

Comment:
Validation will be failed if you set php 7.0 in app.yaml (you cannot provide patch version in this file only the minor one).
The same for 7.1

Reply:
Agree, forgot about this, thank you!